Small Snake Nerfs

jdipalermo27 opened the pull request:
Snake currently has a dilemma where he received no nerfs going from vanilla to hdr and just big buffs to his cqc. While part of his issue revolves around the armor on cipher, that is currently being changed on another pr. This pr is to nerf the kill power and balance the active frames and landing lag on his aerials, but also gives a slight buff to ftilt 2 and removes the ability of sticky c4 to drop off of opponents when hit. Snake will now require his setplay or advantageous positions to net his kills and can no longer press any button to secure that kill.

<==Snake==>
FTILT 2
~[+] KBG: 95 --> 100
DASH ATTACK
Late Hit
~[-] Active Frame Window: 8-17 --> 8-14
NAIR
~[-] Landing lag: 8 --> 12
~[R] FKB: 55 --> 50
Angle change on multihit hitboxes from 367 to 365
Removal of -7 frame hitstun modifiers on multihits
Final Hit
~[-] KBG: 145 --> 120
FAIR
~[-] Landing lag: 13 --> 14
BAIR
Strong Hit
~[R] Angle: 361 --> 50
~[-] KBG: 100 --> 95
Weak Hit
~[-] Damage: 10 --> 5
~[-] Active Frames: 16 --> 9
~[R] Angle: 361 --> 45
UPAIR
Strong Hit (Feet)
~[-] KBG: 100 --> 90
~[+] BKB: 35 --> 45
Strong Hit (Legs)
~[-] KBG: 100 --> 70
Weak Hit
~[-] KBG: 100 --> 80
DAIR
~[-] Landing lag: 10 --> 12
GRENADE
~[-] Snake exclusive hitbox normalized with frame 2
~[-] Hitbox Size: 20 --> 16
C4
~[+] Sticky will no longer fall off from taking damage.
~[-] Sticky will fall off after 20 seconds.

jdipalermo27 committed:
Small Snake Nerfs

<==Snake==>
FTILT 2
~[+] KBG: 95 --> 100
NAIR
~[-] Landing lag: 8 --> 12
Final Hit
~[-] KBG: 145 --> 120
FAIR
~[-] Landing lag: 13 --> 14
BAIR
Strong Hit (feet)
~[R] Angle: 361 --> 50
~[-] KBG: 100 --> 95
Strong Hit (legs)
~[-] KBG: 100 --> 80
Weak Hit
~[-] Damage: 10 --> 5
~[-] Active Frames: 16 --> 9
~[R] Angle: 361 --> 45
UPAIR
Strong Hit (Feet)
~[-] KBG: 100 --> 88
Strong Hit (Legs)
~[-] KBG: 100 --> 70
Weak Hit
~[-] KBG: 100 --> 80
DAIR
~[-] Landing lag: 10 --> 12
C4
~[+] Sticky will no longer fall off.

hdr-balance-backroom reviewed (changes requested):
- Reduce grenade hitbox size and remove/normalize the Snake-exclusive hitbox
- Change neutral air multihits angle to 365 and lower knockback to reduce frame advantage
- C4 should fall off after a consistent 20 seconds
- Reduce late dash attack activity by 3 frames

jdipalermo27 committed:
Balance backroom followup

Reduce activity on dash attack, nair multihit changes, c4 20 seccond timer on sticky falling off

jdipalermo27 commented:
I need to put in the nade changes still.

SkewedAskew commented:
Incorporated into https://github.com/HDR-Development/HewDraw-Remix/pull/1630
